Section 1

ArturMay opened the pull request:
@RuslanFahrenheit, HTML and CSS for the first section have been added

ArturMay committed:
Added HTML and CSS for the first section

ArturMay committed:
Added HTML and CSS for the first Section

RuslanFahrenheit commented on scss/style.scss:
```diff
@@ -0,0 +1,86 @@
+
```
Redundant empty line.

ArturMay replied:
done

RuslanFahrenheit commented on scss/style.scss:
```diff
@@ -0,0 +1,86 @@
+
+.highlighter {
+    font-size: 90px;
+    padding:0 20px;
```
Code style. Check formatting all over the file.

ArturMay replied:
+checked

RuslanFahrenheit commented on scss/style.scss:
```diff
@@ -0,0 +1,86 @@
+	max-width: 1515px;
+	margin: 0 auto;
+	padding: 0 15px;
+    // border: 1px solid rgb(69, 151, 80);
```
Remove commented code.

ArturMay replied:
I will remove it in the end, it helps me

RuslanFahrenheit commented on scss/style.scss:
```diff
@@ -0,0 +1,86 @@
+    color: #fff;
+    font-size: 20px;
+    margin-bottom: 220px;
+}
```
Add a blank line at the end.

ArturMay replied:
done

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
 <body>
-    <!-- Your code goes here -->
-    <!-- Your code goes here -->
+    <header>
```
An entire section is wrapped in `<header>`. Why? Is this really header?

ArturMay replied:
main has been added. Now header contains logo and menu (navigation) only

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
-    <!-- Your code goes here -->
-    <!-- Your code goes here -->
+    <header>
+      <div class="visual-box">
```
What is the meaning of 'visual-box' class name? I think there are more semantically correct options.
E.g. https://www.google.com/search?client=firefox-b-d&q=hero+section

ArturMay replied:
'visual box has been changed to 'hero-section'.
Also, now the hero-section has a min-height of 100vh and big margins have been removed.

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
+          <div class="header-wrap">
+            <div class="logo">
+              <a href="#">BLACK & WHITE</a>
+            </div>
```
It's a bad idea to use text as a logo. It should always looks the same. Imagine fonts not loading. Always use image as a logo.
E.g. https://www.adidas.com/us 
They don't type 'ADIDAS'. It's an image.

ArturMay replied:
Text in the logo has been changed to image

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
+      </div>
+    </header>
 </body>
 </html>
```
Add blank line.

ArturMay replied:
done. it was the hardest one :)

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
+          <h1>BLACK<span class="highlighter">&</span>WHITE</h1> 
+          <p class="company-slogan">An Original Brand Since 1965</p>
+          <a href="#" class="btn">
+            <span>Show More</span>
```
What is the purpose of `span` inside the link tag?

ArturMay replied:
For making bigger font-size to the symbol "&"

RuslanFahrenheit commented on index.html:
```diff
@@ -5,9 +5,32 @@
+              </ul>
+            </nav>
+          </div> 
+          <h1>BLACK<span class="highlighter">&</span>WHITE</h1> 
```
I suppose "highlighting" is more about making something stand out with color.
Not critical, but think about more semantically correct name.

ArturMay replied:
Changed to "enlarger"

RuslanFahrenheit reviewed (changes requested):
1. Why do you have two identical commits: "Added HTML and CSS for the first section"? First commit is about adding configuration for processor. Describe your commits carefully.
Also read [Как писать сообщения коммитов в Git](https://medium.com/grisme/%D0%BA%D0%B0%D0%BA-%D0%BF%D0%B8%D1%81%D0%B0%D1%82%D1%8C-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D1%8F-%D0%BA%D0%BE%D0%BC%D0%BC%D0%B8%D1%82%D0%BE%D0%B2-%D0%B2-git-9ed19ebc5ebf).
2. I don't see typography styles.
3. The section should be full-height on all screens. Don't use big margin values for that. 
4. Don't write styles in one file. Read:
- https://qna.habr.com/q/741337
- http://prgssr.ru/development/estetichnyj-sass-chast-1-arhitektura-i-organizaciya.html#heading-section

ArturMay committed:
fixed several issues, still need to fix

ArturMay committed:
Devide styles into separate files for styles

Devide header and main part

Make the first section full screen by adding 100vh

Remove big values of margin

Add variables for text and hover color

Add link for fonts

Fix other issues as per comments in the first pull request

ArturMay committed:
Update font link according to the task

Add font family for body

ArturMay committed:
Use @include and @extend for button in

the separate files. Does not work

"Undifined variable"

Something wrong with @extend %button

RuslanFahrenheit commented:
TODO:
- style.scss - only imports here;
- code style;
- remove %button extender;
- add class to `<header>`.
